lib/addons/cloudwatch-logs/index.ts

```diff
@@ -0,0 +1,110 @@
+import { PolicyDocument } from 'aws-cdk-lib/aws-iam';
+import { Construct } from "constructs";
+import * as iam from "aws-cdk-lib/aws-iam";
+import { setPath } from "../../utils";
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import { ClusterInfo, Values } from "../../spi/types";
+import { createNamespace } from "../../utils/namespace-utils";
+import { getCloudWatchLogsPolicyDocument } from "./iam-policy";
+
+/**
+ * Configuration options for the FluentBit add-on.
+ */
+export interface CloudWatchLogsAddonProps extends HelmAddOnUserProps {
```

Comment:
Are there any performance settings that we need to expose? I recall this particular component becoming a bottleneck in large clusters. 
We don't necessarily have to expose all the performance controls. At the very least we should honor any values that customers provide. If we can add a paragraph to the doc about performance tuning that would be awesome (e.g. with a reference to public docs on fluentbit performance tuning).

Reply:
?

Reply:
@shapirov103 I did add a section to doc page on how to do performance tuning. We dont want to support lot of parameters for this but the recommendation will help customers to pass values to achieve it. Please check